src/main/java/exception/JarvisException.java

```diff
@@ -0,0 +1,11 @@
+package exception;
+
+public class JarvisException extends Exception {
+    public JarvisException() {
+        super();
+    }
+
+    public JarvisException(String message) {
```

Comment:
Very clean and good use of inheritance, but might confuse other developers on what error this exception is supposed to handle/catch so maybe JavaDoc comments might help?